capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -0,0 +1,33 @@
+package com.google.univiz;
+
+/**
+ * CarnegieSizeDegree is a class that holds the enum for the range of values for a school's carnegie
+ * size degree.
+ */
+public enum CarnegieSizeDegree {
+  NOT_AVAILABLE("Degree unavailable"),
+  SIZE1("Two-year college, very small"),
```

Comment:
Any reason not to use more descriptive names for these enum constants? For example `SIZE1` could be renamed to `TWO_YEAR_COLLEGE`.